test/webrtc_test.rb

```diff
@@ -0,0 +1,236 @@
+# frozen_string_literal: true
+
+using_task_library "comms_webrtc"
+import_types_from "iodrivers_base"
+import_types_from "base"
+
+describe OroGen.comms_webrtc.Task do
+    run_live
+
+    attr_reader :task1, :task2, :rustysignal_pid
+
+    before do
+        timeout = Time.now + 1
+        while Time.now < timeout
+            begin
+                @rustysignal_pid = Process.spawn("rustysignal", "127.0.0.1:3012")
+                break
+            rescue Errno::ECONNREFUSED
+                sleep(0.01)
+            end
+        end
+        if @rustysignal_pid.nil?
+            raise "Timeout to start rustysignal"
+        end
```

Comment:
This does not do anything of the stuff it's supposed to do.

- what does Process.spawn does ?
- why would you get Errno::ECONNREFUSED ?
- why would you get a nil @rustysignal_pid ?

Reply:
The nil check actually it was tied by a misunderstanding and the wrong use of the logic...